### PR TITLE
Added build script for onnxruntime v1.20.0 and updated build_info.json

### PR DESCRIPTION
**Pull request by puneetsharma21**
## Checklist

- [ ] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [ ] Have you validated script on UBI 9 container
- [ ] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: o/onnxruntime/onnxruntime_v1.20.0_ubi_9_3.sh [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+yum install -y gcc-toolset-13-gcc gcc-toolset-13-gcc-c++
+source /opt/rh/gcc-toolset-13/enable
+
+# Cloning the repository 
+git clone $PACKAGE_URL
+cd ${PACKAGE_NAME}
+git checkout $PACKAGE_VERSION
+
+python3.11 -m venv myenv
+source myenv/bin/activate
```
Comment (shubham-dayma-ibm) [inline o/onnxruntime/onnxruntime_v1.20.0_ubi_9_3.sh:66]: No need to create venv
Use just `python` and `pip`, rather using `python3`, `pip3` or something else, to reduce future reworks

File: o/onnxruntime/onnxruntime_v1.20.0_ubi_9_3.sh [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+# Default tag onnxruntime
+if [ -z "$1" ]; then
+  export PACKAGE_VERSION="v1.20.0"
+else
+  export PACKAGE_VERSION="$1"
+fi
+
+# install tools and dependent packages
+yum install -y git gcc-c++ make wget java-11-openjdk-devel  openssl-devel bzip2 zip unzip yum-utils clang clang-devel clang-libs python3.11 python3.11-devel patch
```
Comment (shubham-dayma-ibm) [inline o/onnxruntime/onnxruntime_v1.20.0_ubi_9_3.sh:34]: Remove python installation instructions

**Review by shubham-dayma-ibm — CHANGES_REQUESTED** [review]
@puneetsharma21  We already have build-script of [onnxruntime](https://github.com/ppc64le/build-scripts/blob/master/o/onnxruntime/onnxruntime_ubi_9_3.sh). Have you tried that script with you version? Do we really require new build-script for your version?

File: o/onnxruntime/onnxruntime_v1.20.0_ubi_9_3.sh [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+git checkout $PACKAGE_VERSION
+
+# Build and test package
+if !(./build.sh --allow_running_as_root --compile_no_warning_as_error --config Release) ; then
+    echo "------------------$PACKAGE_NAME:build_fails-------------------------------------"
+    echo "$PACKAGE_URL $PACKAGE_NAME"
+    echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | GitHub | Fail |  Build_Fails"
+    exit 1
+else
+    echo "------------------$PACKAGE_NAME:build_&_test_both_success-------------------------"
```
[the remark this answers is not in the document]
Comment (puneetsharma21) [reply o/onnxruntime/onnxruntime_v1.20.0_ubi_9_3.sh:61]: @npanpaliya tests are executing here in build.sh file
`./build.sh --allow_running_as_root --compile_no_warning_as_error --config Release`

**Review by shubham-dayma-ibm — APPROVED** [review]
LGTM

**Pull request merged**